Duo_S/BuildRoot/README.md

```diff
@@ -11,11 +11,11 @@
 ### Hardware Information
 
 - Milk-V Duo S (512M, SG2000)
-- One USB Power Adapter
-- One USB-A to C or USB C to C Cable for powering the development board
-- One microSD Card
-- One USB Card Reader
-- One USB to UART Debugger (e.g., CP2102, FT2232, etc., take note not to use CH340/341 series as they may cause garbled text)
+- A USB Power Adapter
+- A USB-A to C or USB C to C Cable for powering the development board
+- A microSD Card
+- A USB Card Reader
+- A USB to UART Debugger (e.g., CP2102, FT2232, etc., take note not to use CH340/341 series as they may cause garbled text)
```

Comment:
take note... -> be aware that CH340/341 series will cause garbled text output, DO NOT USE